### PR TITLE
Arguments to "flutter create" command

### PR DESCRIPTION
**Pull request by davidgaspardev**
Added more information about the possibilities of the flutter create command.

### CODE REVIEW

**Comment by davidgaspardev** [comment]
@googlebot I signed it!

File: src/docs/get-started/test-drive/_terminal.md [hunk 1]
```diff
@@ -9,6 +9,13 @@ $ flutter create myapp
 $ cd myapp
 ```
 
+It is also possible to pass more arguments to `flutter create`, such as the project name (*pubspec.yml*), the oraganization or the programming language used for the native platform:
```
Comment (sfshaza2) [inline src/docs/get-started/test-drive/_terminal.md:12]:
```suggestion
It is also possible to pass other arguments to `flutter create`,
such as the project name (*pubspec.yml*), the organization name,
or to specify the programming language used for the native platform:
```

**Review by sfshaza2 — APPROVED** [review]
I made some minor changes to the proposed text, and shortened the lines. LGTM

**Comment by domesticmouse** [comment]
trailing LGTM

**Pull request merged**